### DISCUSSION TITLE
on after email change hook

### DISCUSSION THREAD

**Discussion by ricardoorfao** [fastapi-users/fastapi-users#930] [Ideas]
When users want to change their email, it is good practice to send an activation link to the new address and not switch emails until the new one is activated. This will ensure that the user has entered the correct email address. I was wondering if we could have an `on_after_email_change` hook to use in this use case.

**Comment by frankie567** [comment]
Probably a good approach is to use the `is_verified` flag and related router for this: https://fastapi-users.github.io/fastapi-users/configuration/routers/verify/

**Reply by ricardoorfao** [reply]
Makes sense. Thank you
